packages/wrangler/src/bundle.ts

```diff
@@ -235,24 +249,37 @@ export async function bundleWorker(
 
 	// At this point, inputEntry points to the entry point we want to build.
 
-	const inject: string[] = [];
+	const inject: string[] = injectOption ?? [];
 	if (checkFetch) inject.push(checkedFetchFileToInject);
 	if (experimentalLocalStubCache) {
 		inject.push(
 			path.resolve(getBasePath(), "templates/experimental-local-cache-stubs.js")
 		);
 	}
 
+	const outFilepath = isOutfile
+		? path.isAbsolute(destination)
+			? destination
+			: entry.directory.endsWith("functions")
+			? path.join(entry.directory, "..", destination)
+			: path.join(entry.directory, destination)
```

Comment:
Triple ternary monkaS, can we pls avoid that? Not very readable, some ifs or something would be nicer 🙃